### ISSUE TITLE
iOS app should build in CI

### ISSUE THREAD

**Issue by holmesworcester** [TryQuiet/quiet#1175]
Once we can build iOS locally, we should do the work to get it to build in CI.

**Comment by siepra** [comment]
Blocked by #1170

**Comment by siepra** [comment]
Although CI passes, notarytool complains about submitted .ipa file

**Comment by siepra** [comment]
I added a step for uploading .ipa file as an artifact. It can later be submitted to App Store Connect using altool.
Notarytool still causes problems but it may be rational to postpone it until altool is deprecated and more feedback on notarytool is present in the internet

**Comment by kowalski** [comment]
@siepra I think this is donne, right?

**Comment by siepra** [comment]
It's half done. I bringed back altool as notarytool seems not to work properly but we'll have to use it eventually
